vfs/vfscache/item.go

```diff
@@ -429,6 +460,7 @@ func (item *Item) Open(o fs.Object) (err error) {
 
 	fd, err := file.OpenFile(osPath, os.O_RDWR, 0600)
 	if err != nil {
+		item.opens--
```

Comment:
None of the other error handling does `item.opens--` so this sticks out. Maybe they all should, I'm not sure...

Reply:
I will check all errors under item.Open, item.Close, and item.Reset to see if they adjust item.opens correctly and discuss with you upon questions.